package.json

```diff
@@ -85,7 +85,7 @@
     "prettier": "prettier --write \"./**/*.{js,md,yml,ts,tsx}\"",
     "format-check": "prettier --list-different \"./**/*.{js,md,yml,ts,tsx}\"",
     "update-lock": "npx yarn-deduplicate",
-    "docker-setup-android": "docker pull reactnativecommunity/react-native-android:6.1",
+    "docker-setup-android": "docker pull reactnativecommunity/react-native-android:latest",
```

Comment:
headsup, this should be 6.2 not latest

Reply:
```suggestion
    "docker-setup-android": "docker pull reactnativecommunity/react-native-android:6.2",
```

Reply:
I intentionally changed to `latest` so we don't have to change it over and over.

But as this file is the `package.json` for a given version of RN, we probably want to keep it aligned on a given release of the Docker image

Reply:
I've updated it